src/Config/SDK/Parser/AttributesParser.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Config\SDK\Parser;
+
+use OpenTelemetry\SDK\Common\Configuration\Parser\MapParser;
+
+class AttributesParser
+{
+    public static function parseAttributesList(?string $list): array
+    {
+        return MapParser::parse($list);
+    }
+
+    /**
+     * @param array{
+     *     array{
+     *         name: string,
+     *         value: mixed,
+     *         type: ?string,
+     *     }
+     * } $input
+     */
+    public static function parseAttributes(array $input): array
+    {
+        $attributes = [];
+        foreach ($input as $attr) {
+            $attributes[$attr['name']] = $attr['value'];
+        }
+
+        return $attributes;
```

Comment:
Should filter `null` values.

Reply:
Updated to use `array_column` + remove nulls from arrays, and remove keys with null scalar values